tests/cephfs/cephfs_utils.py

```diff
@@ -56,16 +56,29 @@ def get_clients(self, build):
                 out, rc = node.exec_command(cmd='sudo rpm -qa | grep -w attr')
                 output = out.read().decode()
                 output.split()
+                distro, rc =node.exec_command(cmd='sudo lsb_release -a')
+                distro_ver = distro.read().decode()
+                distro_ver.split()
+                if '7.' in distro_ver:
+                    node.exec_command(
+                            cmd='sudo curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py --insecure',
+                                long_running=True)
+                    node.exec_command(
+                            cmd='sudo python get-pip.py --trusted-host files.pythonhosted.org '
+                                '--trusted-host pypi.org --trusted-host pypi.python.org',
+                                long_running=True)
+                else:
+                    node.exec_command(cmd='sudo yum install -y python2-pip')
                 if 'attr' not in output:
                     node.exec_command(cmd='sudo yum install -y attr')
                 node.exec_command(cmd='sudo yum install -y gcc python2-devel', check_ec=False)
 
                 out, rc = node.exec_command(
-                    cmd='sudo pip  list --format=columns')
+                    cmd='sudo pip2  list --format=columns')
```

Comment:
i would suggest stop using python2 code here. and start using python3. convert the existing scripts to py3. we wont be needing to use py2 anymore. if python3 is not installed in rhel 7, pls install